### DISCUSSION TITLE
Define your own Chat Pipelines

### DISCUSSION THREAD

**Discussion by hra42** [pieces-app/support#323] [Ideas]
I'd like to build my own Chat Pipelines inside PfD.

For example: I have to do daily time and task tracking, live context helps with that, so I have to build a prompt that does this for me. With a Chat Pipeline, I could shorten the used prompt and go even deeper with the tasks and programs I'd like to track.

**Comment by MovGP0** [comment]
Are predefined System Prompts sufficent, like OpenAI's [GPTs](https://openai.com/index/introducing-gpts/)?

Or do we need support for more complex Agentic Workflows like in [LangChain](https://www.langchain.com/) or [promt flow](https://microsoft.github.io/promptflow/)?

If it's the later, do we require a graphical editor like [LangGraph](https://www.langchain.com/langgraph) to manage those flows?
